docs(react): update react tutorial to match the latest version of Nx (#7639)
nrwl/nx: docs/react/tutorial/05-add-node-app.md

});
server.on('error', console.error);
```

**Now run `npx nx serve api` to run the api server**

> Finally, refresh the front-end to hit the server. You will see that the React app is now able to fetch and create todos by calling the API.

Reviewer: ```suggestion
Refresh the application in the browser. The React app is now able to fetch and create todos by calling the API.
```
